pkg/rke/templates/coredns_v1.8.3-rancher3.go

```diff
@@ -0,0 +1,370 @@
+package templates
+
+/*
+Rancher Changelog:
+- coredns deployment: liveness and readiness probe configuration
+- coredns deployment: drop the replica
+- coredns deployment: replace the usage of `beta.kubernetes.io/os: linux` with `kubernetes.io/os: linux`
+- coredns deployment: replace the seccomp annotation with seccompProfile
+- coredns-autoscaler deployment: liveness and readiness probe configuration
+- coredns-autoscaler deployment: replace the usage of `beta.kubernetes.io/os: linux` with `kubernetes.io/os: linux`
```

Comment:
Are all these net new changes for `-rancher3` template or carried forward from previous template?

Reply:
yes

Reply:
upgrade checks and provisioning checks are done for 2.8 and 2.7 already with this template.. waiting for this to be merged before QA picks it up for 2.9. This is just  a forward port of the same issue.